scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -50,7 +50,7 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
   def parseStat(): Stat = parseRule(parser => parser.statSeq[Stat] {
     case token if token.is[`import`] => importStmt()
     case token if token.is[`package `] => packageOrPackageObjectDef()
-    case token if token.is[TokenClass.DefIntro] => nonLocalDefOrDcl()
+    case token if token.is[TokenClass.DefIntro] || token.is[Ellipsis] => nonLocalDefOrDcl()
```

Comment:
This is the only case when `Ellipsis` could (possibly) be merged in `TokenClass`. But that would lead to unnecessary checks inside `DefIntro`, because other parsing situations would become broken with naively added `Ellipsis`. So, I decided to put simplest check here, instead of several complicated in `DefIntro`.